server/athenian/api/precompute/__main__.py

```diff
@@ -148,4 +141,11 @@ async def async_entry():
 
 
 if __name__ == "__main__":
-    exit(_main())
+    athenian.api.db._testing = True
+    patch_pandas()
+    log = logging.getLogger("precomputer")
+    args = _parse_args()
+    setup_context(log)
+
+    with sentry_sdk.start_transaction(name="precomputer[accounts]", op="precomputer[accounts]"):
+        exit(_main(args, log))
```

Comment:
We shouldn't execute code outside of a function. Let's factor another one. E.g. `_main` and `_execute_command`